.buildkite/pipeline.yml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+steps:
+  - label: "build-gnu-x86"
```

Comment:
I think you still need to keep these steps:
```yml
 - label: "build-gnu-x86-elf"
    commands:
     - cargo build --release --features elf
    retry:
      automatic: false
    agents:
      platform: x86_64.metal
    plugins:
      - docker#v3.0.1:
          image: "fandree/rust-vmm-dev"
          always-pull: true

  - label: "unittests-gnu-x86-elf"
    commands:
     - cargo test --features elf
    retry:
      automatic: false
    agents:
      platform: x86_64.metal
    plugins:
      - docker#v3.0.1:
          privileged: true
          image: "fandree/rust-vmm-dev"

  - label: "build-gnu-x86-bzImage"
    commands:
     - cargo build --release --features bzImage
    retry:
      automatic: false
    agents:
      platform: x86_64.metal
    plugins:
      - docker#v3.0.1:
          image: "fandree/rust-vmm-dev"
          always-pull: true

  - label: "unittests-gnu-x86-bzImage"
    commands:
     - cargo test --features bzImage
    retry:
      automatic: false
    agents:
      platform: x86_64.metal
    plugins:
      - docker#v3.0.1:
          privileged: true
          image: "fandree/rust-vmm-dev"
```
because you want to check that the build passes with individual features as well. Actually, since elf is the default feature now, you can skip the elf build step and only add the bzImage one.

Reply:
Done.
I did not realize we switched to using `ELF` as the default feature.